Makefile

```diff
@@ -14,6 +14,11 @@ GCC = g++
 endif
 LD = $(GCC)
 
+# Clang-specific flags
+ifeq ($(GCC), clang++)
```

Comment:
Won't work for me as I use clang++ with a path prefix and sometimes disguised as g++. I don't think anyone except me needs to build non-RELEASE with g++, so could you move these back to the main CFLAGs?

Reply:
Alright, I dropped the second commit.
There are too many issues currently when trying to build non-`RELEASE` with GCC anyway.